### ISSUE TITLE
Design Intent - Accessibility Feedback - Transition Experience, tbd, V1 Transition Experience

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#84075]
## Next Steps for the VFS team

- [ ] **Assign** this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] **Comment** on this ticket:
  - If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members`.
- [ ] **Close the ticket** when all feedback has been addressed.

## Thoughts/questions

- I really like the direction this is taking! I think you're definitely on the right track.

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform Governance reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- [ ] **Consider:** Looking at the sequence in which you're asking for information, think about the placement of the time-sensitive benefits. Determining what should appear there may make more sense after asking for service history (calculating from the dates provided), instead of asking about when they served in the identity questions. 
   
   But asking in the identity questions isn't necessarily bad, if it makes more sense to do it then. It may come down to how granular you need the responses to be in order to show the time-sensitive benefits. If asking about service history in the identity questions, be careful about cognitive load when making users do too much calendar math -- eg. "Did you start your service after January 1, 2000?" is probably an easier question to answer than "How many years ago did you start your service?"
- [ ] **Must:** Reminder to be careful about material honesty -- if it navigates you to another page, it should be coded as a link and it should look like a link. If it performs an action, it should be coded as a button and it should look like a button. I'm thinking in particular about the "Share Results" link on the ["Your benefits and resources" page](https://www.figma.com/design/Y72reZ0Qt9xteezdnLbBDW/Wireframes-for-v1?node-id=70-5295&t=FkgFSjGY93GYG0oB-0), which opens a modal and should be a button.
- [ ] **Must:** As you refine the ["Your benefits and resources" page](https://www.figma.com/design/Y72reZ0Qt9xteezdnLbBDW/Wireframes-for-v1?node-id=70-5295&t=FkgFSjGY93GYG0oB-0), be careful about placing elements along the right-hand side of the screen. Some low vision users use screen magnification software (or even physical devices mounted to their screens) that magnify a portion of the page, as opposed to using something like browser zoom. If you're magnifying the left-hand side of the screen and following the flow of content down, you may not notice elements floating to the right unless there's some visual cue to point you over there to look. I'm thinking "Share Results" and "Available now" / "Available in 60 days" / etc are likely to be missed.
- [ ] **Must:** When filtering results on the ["Your benefits and resources" page](https://www.figma.com/design/Y72reZ0Qt9xteezdnLbBDW/Wireframes-for-v1?node-id=70-5295&t=FkgFSjGY93GYG0oB-0), a few items that I strongly recommend be included in the interaction:
   - The action to filter should be triggered by a button --- and selecting options from the dropdowns shouldn't trigger anything. Keyboard users arrowing down through a `select` element may accidentally trigger the filter action if you have an event listener on the dropdowns themselves, and if that moves keyboard focus to the results it could cause you to change contexts before you're ready.
   - Some kind of status text should display on screen between the filters and the results, eg. "Displaying X items, filtered by Y and Z." That kind of message helps reduce cognitive load by showing users what's currently displayed. (It also helps out in the next bullet point too!)
   - After triggering the filter action, keyboard focus should move to the status text. This ensures that screen reader users hear the status of the filter immediately after the filter action has taken place, verifying that the action was successful. Without that message, screen reader users may not hear anything and may have to guess-and-check to see if the filter actually happened.

## Governance team actions
- [x] Format feedback as individual tasks (check boxes)
- [x] Assign this ticket to the VFS team member that opened the Slack request
- [x] Add the VFS team product label
- [x] Add the VFS team the feature label (if applicable)
- [x] Add the touchpoint labels
- [x] Add the practice area labels
- [x] Add the Collaboration Cycle initiative milestone

**Comment by nkontrabecki** [comment]
All items addressed. Closing ticket
